src/libopensc/card-piv.c

```diff
@@ -3029,10 +3164,16 @@ static int piv_match_card_continued(sc_card_t *card)
 				}
 			}
 		}
+		sc_debug(card->ctx,SC_LOG_DEBUG_MATCH, "PIV_MATCH card->type:%d type:%d r:%d\n", card->type, type, r);
+
 		if (type == -1)
```

Comment:
Missing braces around the following block

Reply:
Fixed added brackets.